internal/build/lifecycle_execution.go

```diff
@@ -181,6 +182,10 @@ func (l *LifecycleExecution) Create(ctx context.Context, publish bool, dockerHos
 		flags = append(flags, "-skip-restore")
 	}
 
+	if l.opts.GID >= 0 {
```

Comment:
Could we pull this `0` inot a constant, so it's not a magic number? For all of the GID comparisons in this file

Reply:
Sure. @dfreilich